OctoKit/Issue.swift

```diff
@@ -0,0 +1,151 @@
+import Foundation
+import RequestKit
+
+// MARK: model
+
+@objc public class PullRequest: NSObject {
+    public var url: String?
+    public var htmlUrl: String?
+    public var diffUrl: String?
+    public var patchUrl: String?
+    
+    public init(_ json: [String: AnyObject]) {
+        url = json["url"] as? String
+        htmlUrl = json["html_url"] as? String
+        diffUrl = json["diff_url"] as? String
+        patchUrl = json["patch_url"] as? String
+    }
+}
+
+@objc public class Issue: NSObject {
+    public let id: Int
+    public var url: String?
+    public var htmlUrl: String?
+    public var number: Int?
+    public var title: String?
+    public var body: String?
+    public var state: String?
+    public var user: User
+    public var assignee: User?
+    public var pullRequest: PullRequest?
```

Comment:
I am not sold on this tbh. I would treat the fact that PullRequests and issues are nearly identical as a coincidence and treat them as separate things and therefore separate models.